### PR TITLE
Package soundtouch.0.1.9

### PR DESCRIPTION
**Pull request by toots**
### `soundtouch.0.1.9`
Bindings for the soundtouch library which provides functions for changing pitch and timestretching audio data

---
* Homepage: https://github.com/savonet/ocaml-soundtouch
* Source repo: git+https://github.com/savonet/ocaml-soundtouch.git
* Bug tracker: https://github.com/savonet/ocaml-soundtouch/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

**Comment by toots** [comment]
Ok, sorry for the delay, there were some complications due to the fact that soundtouch's `pkg-config` package has different names with some distributions. This looks good now.

File: packages/soundtouch/soundtouch.0.1.9/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+synopsis:
+  "Bindings for the soundtouch library which provides functions for changing pitch and timestretching audio data"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "LGPL-2.1"
+homepage: "https://github.com/savonet/ocaml-soundtouch"
+bug-reports: "https://github.com/savonet/ocaml-soundtouch/issues"
+depends: [
+  "ocaml"
+  "dune" {> "2.0"}
```
Comment (mseri) [inline packages/soundtouch/soundtouch.0.1.9/opam:11]:
```suggestion
  "dune" {>= "2.0"}
```
Looks like 2.0 should be fine

**Comment by toots** [comment]
Thanks @mseri

**Comment by mseri** [comment]
Thabo you!

**Pull request merged**